bescort.lic

```diff
@@ -831,18 +831,18 @@ class Bescort
     fix_standing
     fput('push boulder')
     case waitfor(/^At the bottom of the hollow/, /^You stop pushing/, /blocking all access to a low tunnel\.$/)
-    when /You stop pushing/, /blocking all access to a low tunnel/
+    when /^You stop pushing/, /blocking all access to a low tunnel\.$/
       push_boulder
     end
   end
   
   def gos_tunnel
-    fput('kneel')
+    fput('kneel', 'You kneel down upon the ground')
```

Comment:
💡 Can this use `bput` instead?

Reply:
aka please consider switching it over in the future even though its merged.

Reply:
you might have meant to switch it to bput there, fput doesnt take args like that

Reply:
`fput` does take args, similar to `bput`, it's just less reliable.

Reply:
https://github.com/rcuhljr/dr-lich/blob/master/lich.rbw#L5700